### DISCUSSION TITLE
Any advantage to using the Pico W model with wifi?

### DISCUSSION THREAD

**Discussion by darrenoc3** [OpenStickFoundation/GP2040-CE#135] [Q&A] [answered]
Are there are any plans for GP2040-CE to make use of the onboard wifi of the Pico W model in future (e.g. to run the web configurator on your phone), or is it better to stick to the slightly cheaper non-wireless model?

**Comment by TheTrainGoes** [comment, marked as answer]
Hey @darrenoc3 ,

Sorry about the late reply on this.  

Currently there are no plans to utilize the Pico W on GP2040-CE.  

Our current roadmap is get as many features and bugs take care of as possible.  Once that all happens we may take a look at the Pico W and what can be done with it.  

As always, you are welcome to play around with it and submit PRs with anything you build around it :)
